.github/workflows/client.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Build
         run: |
           cd client/
-          yarn build:docker
+          yarn build#:docker
```

Comment:
typo

Reply:
Yeah I need putting only `yarn build` because that's all that's defined in package.json for the client